Hero markup/styles

UiMi17 opened the pull request:
Додав розмітку та стилі для hero + пофіксив 2 зображення (спец. помилка з ретіною).

UiMi17 committed:
Basic HTML hero markup

Commit:
Merge branch 'main' into um-hero

UiMi17 committed:
Merge branch 'main' into um-hero

UiMi17 committed:
Hero markup/styles

Додана розмітка та стилі для hero, прибрані font-face шрифти

UiMi17 committed:
Merge branch 'main' into um-hero

UiMi17 committed:
Revert "Merge branch 'main' into um-hero"

This reverts commit c37cba7f7294f3169c0221984742acb0ee6af73b, reversing
changes made to 908b63d9ec7bb1357ad94e6170dfdd1c5675785a.

UiMi17 committed:
Deleted

UiMi17 committed:
Main merged

UiMi17 committed:
Added basic markup/styles

oasi911 commented:
В общем все супер, нашел только 2 небольших момента
Не хватает полоски над буквой Е в десктоп версии
Иконки кнопки shop now по макету немного отличаются на разных размерах экрана

UiMi17 committed:
Fix

Зробив іконки кнопки для hero правильних розмірів, додав верхній бар над останньою літерою тексту h1 "Anemone".
